folder_structure.yaml

```diff
@@ -0,0 +1,23 @@
+Project:
+  Snakemake:
+    config: #configfile.yaml
+    envs: #rules_envs.yaml
+    scripts: #all_python.py
+    rules: #rules.smk
+    logs: #rule.log
+    #snakefile
+  vcfDir:
+    #to_split.vcf.gz
+    #Chr_final.vcf.gz
+    #Chr_final.vcf.gz.csi
+    Chr: #raw.vcf.gz
+    combined: #to_split.vcf.gz (after split)
+  mapsDir: #genetic.map
+  ancestral_allele: #ancestral.txt
+  Tsinfer:
+    #chr_ancestral.vcf.gz
+    #chr_ancestral.vcf.gz.csi
+    files: #metadata.txt
```

Comment:
So the metaData will be an input provided by the user - so it's probably best that all input files come from the one place (that the user defines). Or multiple places - but it's wherever the user keeps them. We must probably create this structure just for the files created in the pipeline, hence the output.